test/TEST-50-MULTINIC/test.sh

```diff
@@ -365,7 +362,7 @@ test_setup() {
     # Make server's dracut image
     "$basedir"/dracut.sh -l -i "$TESTDIR"/overlay / \
         -m "dash rootfs-block debug kernel-modules watchdog qemu network network-legacy" \
-        -d "af_packet piix ide-gd_mod ata_piix ext3 sd_mod nfsv2 nfsv3 nfsv4 nfs_acl nfs_layout_nfsv41_files nfsd e1000 i6300esb ib700wdt" \
```

Comment:
Multiple `modprobe: FATAL: Module ib700wdt not found in directory /lib/modules/6.1.9-200.fc37.x86_64`:

https://github.com/dracutdevs/dracut/actions/runs/4076254978/jobs/7075949251#step:4:386

`ib700wdt` module required by: https://github.com/dracutdevs/dracut/blob/master/modules.d/04watchdog/watchdog.sh

Reply:
> Not sure if this is in scope for this PR, but we could consider removing ib700wdt from watchdog module.

Indeed, out of the scope of this PR.

Reply:
Okay, I'll file a separate MR for the watchdog changes.